index.rst

```diff
@@ -79,7 +81,7 @@ However, the possible failures can be divided into a handful of cases:
 
 In general, the Rubin Observatory project has followed a policy of processing data at most once, not at least once.
```

Comment:
I'm not so sure about this as a general principle.  We calculate overlaps in DRP, and we will likely even process the same tract twice at different locations to ensure that processing is working correctly.  Butler ingest, EFD transformation, and EFD summarization are meant to be idempotent.

But I agree with the statement below about Prompt Processing.

Reply:
Ok, will remove. For the record, though, I'm pretty sure this sentence was originally quoting _you_.